### ISSUE TITLE
support groupBy query

### ISSUE THREAD

**Issue by Bamieh** [masumsoft/express-cassandra#169]
Hello 👋 

Please add support for the `GROUP BY` cql syntax in the queries. I can contribute with a little bit of guidance if needed.

example cql query:

```cql
select 
	flow_id,
	flow_title,
	SUM(combination_successes) as successes,
	SUM(combination_skipped) as skipped,
	SUM(combination_failures) as failures,
	SUM(combiantion_total) as total,
	total_number_of_flow_combinations,
	COUNT(flow_id) as current_number_of_flow_combinations
from flows_by_job_id where job_id = 6426830190969622528 GROUP BY flow_id
```

I was able to translate the query fully to express-cassandra except the `GROUP BY` part.

```js
	models.instance.flowsByJobId.findAsync({
      job_id: models.datatypes.Long.fromString(jobID),
    }, {
      select: [
        'flow_id',
        'flow_title',
        'SUM(combination_successes) as successes',
        'SUM(combination_skipped) as skipped',
        'SUM(combination_failures) as failures',
        'SUM(combiantion_total) as total',
        'total_number_of_flow_combinations',
        'COUNT(flow_id) as current_number_of_flow_combinations',
      ],
    })
```

Am I doing something wrong? I was hoping for a `$groupBy: ["flow_id"]` or something similar under the `select` property.

notice that without the `GROUP BY` i get different results from the query, hence the arouse my need.

Cheers!

**Comment by masumsoft** [comment]
It should be simple to add group by support. If you want to contribute, you may have a look at this method where orderby is implemented:

https://github.com/masumsoft/express-cassandra/blob/master/src/orm/base_model.js#L296

It should be simpler to implement group by then order by, as it does not have any ASC/DESC handling.

**Comment by Cleomir** [comment]
Any progress on this? I also need to implement a query using GROUP BY and I couldn't make it work using `models.instance.Person.execute_query`

**Comment by Bamieh** [comment]
Currently I have no capacity to contribute to the project, but i was able to hack it easily following @masumsoft's instructions in the comment above.

**Comment by masumsoft** [comment]
Group By support published with v2.3.0
